index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rock, Paper, Scissors</title>
+</head>
+
+<body>
+
+<script>
+  
+function computerPlay() {
+    let rand = Math.random();
+    if (rand < 0.34) {
+        return 'rock';
+    }else if(rand <= 0.67) {
+        return 'paper';
+    }else{
+        return 'scissors';
+    }
+}
+
+let playerScore = parseInt(0);
+let computerScore = parseInt(0);
+
+function playRound(playerSelection, computerSelection) {
+if(playerSelection.toLowerCase() === 'rock'){
+    if(computerSelection === 'paper'){
+        computerScore++;       
+    }else if(computerSelection === 'scissors'){
+        playerScore++;       
+    }else{
+        return;
+    }
+}
+
+if(playerSelection.toLowerCase() === 'paper'){
+    if(computerSelection === 'paper'){
+        return;
+    }else if(computerSelection === 'scissors'){
+        computerScore++;
+    }else{
+        playerScore++;
+    }
+}
+
+if(playerSelection.toLowerCase() === 'scissors'){
+    if(computerSelection === 'paper'){
+        playerScore++;
+    }else if(computerSelection === 'scissors'){
+        return;
+    }else{
+        computerScore++;
+    }
+}
+}  
+ 
+function game(){
+for (let gameCount = 0; gameCount !==5; gameCount++) { 
+    let playerSelection = prompt('Write either rock, paper, or scissors to play');   
+    let computerSelection = computerPlay();
+    playRound(playerSelection, computerSelection);  
+}  
+}
+game();
+
+if(playerScore>computerScore){
+    console.log('Computer '+computerScore+' Player '+playerScore+'. You WIN!!');
```

Comment:
Use string interpolation rather than concatenation. i.e. \`Computer  ${computerScore} Player ${playerScore}. You WIN!!'\`